ISISComputingGroup/ibex_gui
# Ticket2587 e4 menu help

### Description of work

Added Help menu options - About, User Manual, and Console log. 

### To test

* Check all Help menu options lead to correct places
* Change the version in e4.client/plugin.xml. Confirm that this is reflected in the about dialog (Note that this is different from the place it is changed in e3)

### Acceptance criteria

*List the acceptance criteria for the PR*

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
